azuread/resource_application.go

```diff
@@ -904,3 +928,31 @@ func adApplicationSetOwnersTo(client graphrbac.ApplicationsClient, ctx context.C
 
 	return nil
 }
+
+func aadApplicationFindByName(client graphrbac.ApplicationsClient, ctx context.Context, name string) (*graphrbac.Application, error) {
```

Comment:
i think this would make sense in the graph helper package as graph.ApplicationFindByName?

Reply:
Yeah absolutely, I'll move them there 👍